templates/404.html

```diff
@@ -16,12 +16,22 @@
                         <h1>404 Not found</h1>
                     </div>
                     {% include "_messages.html" %}
+                    {% if languages %}
+                    <div class="alert alert-danger">
+                        <p>The course is not available in this language.</p>
```

Comment:
These are missing a translation block.

Reply:
For loop doesn't work inside the translation block because "'blocktrans' doesn't allow other block tags (seen 'for lang in languages') inside it". Is it a good solution to wrap only the text part inside translation block?

Reply:
You don't need to translate the for part, only the text before it